paasta_tools/utils.py

```diff
@@ -1700,3 +1701,21 @@ def prompt_pick_one(sequence, choosing):
         sys.exit(1)
     else:
         return result
+
+
+def encode_object_to_utf8(obj):
+    """
+    http://stackoverflow.com/questions/1254454/fastest-way-to-convert-a-dicts-keys-values-from-unicode-to-str#answer-1254499
+    """
+    if isinstance(obj, basestring):
+        return str(obj)
+    elif isinstance(obj, collections.Mapping):
+        return type(obj)(map(encode_object_to_utf8, obj.iteritems()))
+    elif isinstance(obj, collections.Iterable):
+        return type(obj)(map(encode_object_to_utf8, obj))
+    else:
+        return obj
+
+
+def load_json(*args, **kwargs):
+    return encode_object_to_utf8(json.loads(*args, **kwargs))
```

Comment:
this is the opposite of what you usually want to do.  The best practice is to encode / decode from bytes at application boundaries and deal with text inside your application.

With stdlib json at least you can do this with the `encoding=...` parameter in python2 (python3 enforces that you use text internally when dealing with the `json` interface).

This is probably not the right approach as written ;)